utils/events.py

```diff
@@ -135,3 +153,305 @@ def ensure_event_happens(self, target_type, target_id, event_type):
         if len(events) == 0:
             raise AssertionError(
                 'Event {}/{}/{} did not happen.'.format(event_type, target_type, target_id))
+
+
+class EventBuilder(object):
+    """
+    this class just simplifies "expected" event creation.
+
+    Usage:
+        builder = EventBuilder()
+        evt = builder.new_event(target_type='VmOrTemplate',
+                                target_name='my_lovely_vm',
+                                event_type='vm_create')
+    """
+    def __init__(self):
+        self._tool = EventTool(get_or_create_current_appliance())
```

Comment:
I agree. I just wanted to do this easy and dirty and fix it when EventListener is attached to appliance. Fixed